packages/hardhat-chai-matchers/src/internal/changeEtherBalances.ts

```diff
@@ -47,27 +50,29 @@ export function supportChangeEtherBalances(Assertion: Chai.AssertionStatic) {
             actualChanges.forEach((change, i) => {
               if (change.eq(BigNumber.from(balanceChanges[i]))) {
                 lines.push(
-                  `Expected ${
+                  `Expected ether balance of ${
```

Comment:
"the"?

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/2893/commits/b7051adf1189374e1d21605e6a7c42d46f78c887